### PR TITLE
Add comments about our previous work to ML Kit post

### PR DESCRIPTION
**Pull request by mats-claassen**

### CODE REVIEW

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 1]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
 
 ---
 
-
-Google released [ML Kit](https://firebase.google.com/docs/ml-kit/), which is part of Firebase, last month at its I/O 2018.
-ML Kit allows developers to run machine learning models both on Android and iOS using TensorFlow Lite as framework.
+At Xmartlabs we have been working on Machine Learning tasks for a while now.
+Our main focus has been Machine Learning on iOS and we have worked on apps running style transfer, human body pose detection and classifying ultrasound signals from medical devices.
```
Comment (bryant1410) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: At the end: ", to cite some examples."

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 2]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+We started developing [Bender](https://github.com/xmartlabs/Bender) over a year ago, which is a framework we have used to run neural network models fast on iOS.
+When Apple released Core ML, which is also a framework to run ML models on iOS, and it validated our path so far, and we excitedly got deep into it.
```
Comment (bryant1410) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: ~and~ it validated

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 3]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+Now, we also build [Litio](https://litio.ai/), a platform where you can convert your ML models and get a framework ready to be run on a mobile app.
```
Comment (bryant1410) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: build -> have built

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 4]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
 
 ---
 
-
-Google released [ML Kit](https://firebase.google.com/docs/ml-kit/), which is part of Firebase, last month at its I/O 2018.
-ML Kit allows developers to run machine learning models both on Android and iOS using TensorFlow Lite as framework.
+At Xmartlabs we have been working on Machine Learning tasks for a while now.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: At Xmartlabs we have been closely involved in the nascent field of on-device Machine Learning.

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 5]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+Our main focus has been Machine Learning on iOS and we have worked on apps running style transfer, human body pose detection and classifying ultrasound signals from medical devices, to name some examples.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: Our main focus has been Machine Learning on iOS and we have been exploring different use cases including style-transfer, human body pose detection and medical imaging classification.

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 6]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+We started developing [Bender](https://github.com/xmartlabs/Bender) over a year ago, which is a framework we have used to run neural network models fast on iOS.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: Not only we have built some ~cool~ [*almost magical* apps](https://itunes.apple.com/us/app/envision-artistic-filters/id1251483734?mt=8), but we have also rolled our sleeves to create first of its kind development tools: we released [Bender](https://github.com/xmartlabs/Bender) a few days before WWDC 17, and launched [Litio.ai](https://litio.ai/) earlier this year.

[Bender](https://github.com/xmartlabs/Bender) is an open source iOS framework built on top of Metal that allows developers to run ML models with unmatched flexibility and performance. [Litio.ai](https://litio.ai/) brings the power of Bender to the masses by providing a higher level, cross-platform solution for on-device Machine Learning.

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 7]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+When Apple released Core ML, which is also a framework to run ML models on iOS, it validated our path so far, and we excitedly got deep into it.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: Unsurprisingly, Apple and Google have joined the party as well. In this blog post we will introduce and compare their offerings  by going through an example project that you can follow step-by-step.

Apple introduced CoreML, an on-device ML framework for iOS, during WWDC17. It has recently announced an upgrade in the latest edition of their annual developer's conference.

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 8]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+Now, we have also built [Litio](https://litio.ai/), a platform where you can convert your ML models and get a framework ready to be run on a mobile app.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: delete

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 9]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+Google released [ML Kit](https://firebase.google.com/docs/ml-kit/) last month at its I/O 2018, and its purpose is very similar to the one of Litio: facilitating the deployment to mobile.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: Google released [ML Kit](https://firebase.google.com/docs/ml-kit/) last month at its I/O 2018, a high-level solution for cross-platform ML deployment.

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 10]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+This is why we wrote this post, trying out ML Kit on iOS :)
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: delete

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 11]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
+
+ML Kit, which is part of Firebase, allows developers to run machine learning models both on Android and iOS using TensorFlow Lite as framework.
 It includes Machine Learning models for common use cases like recognizing text, detecting faces, labeling images and more.
 You can use these models out of the box without knowing anything about machine learning, which is cool.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: You can use these models out of the box, no Machine Learning background required.

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 12]
```diff
@@ -9,9 +9,15 @@ markdown: redcarpet
...
 For some of these models you can also choose to run them on the device, for more privacy and speed, or in the cloud for more accuracy.
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: For some of these models you can also choose to run them on the device, for more privacy and speed, or in the cloud for higher accuracy.

File: _posts/2018-06-21-ML-Kit-CoreML.md [hunk 13]
```diff
@@ -321,8 +327,9 @@ You can see that in the following image, where Core ML is running during the fir
...
+We saw that it is quite easy to get set up with ML Kit on iOS and while it is still in Beta, and there are some green parts like the error handling, it is very helpful if you want to add machine learning to your apps on iOS and Android. It is also very helpful that it comes with some models out of the box.
 
-We saw that there was a small difference in performance between ML Kit and Core ML but it was not very big for this model (MobileNet). It is possible that for other models this difference might be bigger but it seems reasonable to use ML Kit if the real time performance is not critical.
+We saw that there was a small difference in performance between ML Kit and Core ML, which did not significantly impact this model (MobileNet). It is possible that for other models this difference might be more significant but it seems reasonable to use ML Kit if the real time performance is not critical.
 
-All the code is on this [GitHub repo](https://github.com/xmartlabs/MLKitTest)
+All the code for this post is on this [GitHub repo](https://github.com/xmartlabs/MLKitTest).
+If you have any questions or comments, or need help with a project, don't hesitate to contact us!
```
Comment (agustinh17) [inline _posts/2018-06-21-ML-Kit-CoreML.md]: If you have any comment or question, or need help implementing a breakthrough use case of on-device Machine Learning, don't hesitate to [contact us](https://xmartlabs.com/contact)!

**Pull request merged**